public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ModalAlerts/UnsupportedDataSourcesAlert.tsx

```diff
@@ -33,7 +33,11 @@ export const UnsupportedDataSourcesAlert = ({ unsupportedDataSources }: { unsupp
         )}
       </p>
       <a
-        href="https://grafana.com/docs/grafana/next/dashboards/dashboard-public/"
+        href={
+          config.featureToggles.newDashboardSharingComponent
+            ? 'https://grafana.com/docs/grafana/next/dashboards/share-dashboards-panels/shared-dashboards/'
+            : 'https://grafana.com/docs/grafana/next/dashboards/dashboard-public/'
```

Comment:
Not sure about this link? `https://grafana.com/docs/grafana/next/dashboards/dashboard-public/ link`

Reply:
why?
I think, it should be 
`https://grafana.com/docs/grafana/next/dashboards/dashboard-public/#supported-data-sources` for the old design. What about the new one?

Reply:
I don't know what this component is, so I don't know why this link to the old docs is there. But as long as you're sure it should be there, then it's all good.

Reply:
Ah sorry, I misunderstood. It should be:
`https://grafana.com/docs/grafana/next/dashboards/share-dashboards-panels/shared-dashboards/#supported-data-sources`